### PR TITLE
fix: PRSDM-6828 Add permalink styling to replicate react toaster

### PR DESCRIPTION
**Pull request by christopherbrunsdon**
## Description

* Update styling for layout-base changes to restore permalink
* Adds toaster
* Adds new icon as per figma design - see ticket for figma designs

This PR has dependency on  [layout-base](https://github.com/SPANDigital/presidium-layouts-base/pull/11)

## Issue
- [PRSDM-6828](https://spandigital.atlassian.net/browse/PRSDM-6828)

## Screenshots

See [enterprise](https://github.com/SPANDigital/presidium-js-enterprise/pull/848) PR for video

image

## PR Readiness Checks
- [x] Your PR title conforms to conventional commits `<type>: <jira-ticket-num><title>`, for example: `fix: PRSDM-123 issue with login` with a maximum of 100 characters
- [x] You have performed a self-review of your changes via the GitHub UI
- [x] Comments were added to new code that can not explain itself (see [reference 1](https://bpoplauschi.github.io/2021/01/20/Clean-Code-Comments-by-Uncle-Bob-part-2.html) and [reference 2](https://blog.cleancoder.com/uncle-bob/2017/02/23/NecessaryComments.html))
- [ ] New code adheres to the following quality standards:
  - Function Length ([see reference](https://martinfowler.com/bliki/FunctionLength.html))
  - Meaningful Names ([see reference](https://learning.oreilly.com/library/view/clean-code-a/9780136083238/chapter02.xhtml))
  - DRY ([see reference](https://java-design-patterns.com/principles/#keep-things-dry))
  - YAGNI ([see reference](https://java-design-patterns.com/principles/#yagni))

### CODE REVIEW

**Review by julianbyte — APPROVED** [review]

File: assets/_sass/_structure.scss [hunk 1]
```diff
@@ -156,6 +156,43 @@
...
+
+      .link-icon {
+        display: block;
+        visibility: hidden;
+      }
+    }
+
+    .link-icon {
+      background-image: url("assets/svg/link-icon.svg");
+      fill: #9a9a99;
```
Comment (CharlRitterDev) [inline assets/_sass/_structure.scss:190]: Is there no colour var we can use here? Think we need to start try pulling hardcoded hex values out

File: assets/_sass/_structure.scss [hunk 2]
```diff
@@ -513,3 +542,22 @@
   padding: 10px;
   text-align: center;
 }
+
+.toaster {
```
Comment (CharlRitterDev) [inline assets/_sass/_structure.scss:546]: I think the naming should just be toast 😝

**Comment by CharlRitterDev** [comment]
Also please test that it looks fine on small screens and for long titles

File: assets/_sass/_structure.scss (hunk 1 above)
Comment (christopherbrunsdon) [reply assets/_sass/_structure.scss:190]: Can introduce a variable, this was from the existing .link-icon that had to move up.

**Review by Quantumplate — APPROVED** [review]
Works really nicely! great job 👍🏼

**Pull request merged**